### PR TITLE
apmsoak: enhance logging

### PR DESCRIPTION
**Pull request by endorama**
Increase logs from `apmsoak` tool.

### CODE REVIEW

File: internal/loadgen/eventhandler.go [hunk 1]
```diff
@@ -38,7 +42,34 @@ type EventHandlerParams struct {
...
+	// FIXME: add Rand.
+	enc.AddBool("ignore_errors", e.IgnoreErrors)
+	enc.AddBool("rewrite_ids", e.RewriteIDs)
+	enc.AddBool("rewrite_service_names", e.RewriteServiceNames)
+	enc.AddBool("rewrite_ids", e.RewriteServiceNodeNames)
+	enc.AddBool("rewrite_ids", e.RewriteServiceTargetNames)
+	enc.AddBool("rewrite_ids", e.RewriteSpanNames)
+	enc.AddBool("rewrite_ids", e.RewriteTransactionNames)
+	enc.AddBool("rewrite_ids", e.RewriteTransactionTypes)
+	enc.AddBool("rewrite_ids", e.RewriteTimestamps)
```
Comment (inge4pres) [inline internal/loadgen/eventhandler.go:66]: Did you mean to rename the keys here?
AFAICT we're adding multiple bool entries on the same "rewrite_ids" field?
I am not very familiar with the library so if it's on purpose, feel free to discard the comment

Comment (endorama) [reply internal/loadgen/eventhandler.go:66]: No, this copy paste failure is on me.

File: internal/loadgen/eventhandler/transport.go [hunk 2]
```diff
@@ -49,12 +53,24 @@ func (t *Transport) SendV2Events(ctx context.Context, r io.Reader, ignoreErrs bo
 }
 
 func (t *Transport) sendEvents(req *http.Request, r io.Reader, ignoreErrs bool) error {
+	t.logger.Debug("sending request")
 	res, err := t.client.Do(req)
 	if err != nil {
-		return err
+		return fmt.Errorf("cannot complete request: %w", err)
```
Comment (inge4pres) [inline internal/loadgen/eventhandler/transport.go:59]: Are we then logging this when it's bubbled up?

File: internal/loadgen/eventhandler.go (hunk 1 above)
Comment (endorama) [reply internal/loadgen/eventhandler.go]: Fixed in [`29d86c6` (#47)](https://github.com/elastic/apm-perf/pull/47/commits/29d86c694da798d5f5b6e19bb86f44de9857e027)

File: internal/loadgen/eventhandler/transport.go (hunk 2 above)
Comment (endorama) [reply internal/loadgen/eventhandler/transport.go:59]: yes, errors bubble up until they are logged in `apmsoak/cmd/run.go`, https://github.com/elastic/apm-perf/blob/3a6d6182717a2bfeff355eebeec1c62b7f7d12a8/cmd/apmsoak/run.go#L73

**Pull request merged**